website/docs/foundations/typography/partials/code/how-to-use.md

```diff
@@ -1,10 +1,20 @@
 ## How to use these styles
 
-While we do offer design tokens for typography, we recommend using the predefined CSS helper classes.
+We offer different ways to apply typography styles to UI elements:
+
+- [Text helper component](#text-component) - This is the **preferred way** to apply HDS text styles to **HTML elements**.
+- [CSS helper classes](#css-helper-classes) - This can be used to apply styles to **Ember components** (or as fallback when using the `Text` helper component is not possible).
```

Comment:
```suggestion
- [CSS helper classes](#css-helper-classes): This can be used to apply styles to **Ember components** (or as fallback when using the `Text` helper component is not possible).
```